proximo/proximo_url.go

```diff
@@ -66,17 +66,15 @@ func newProximoSource(u *url.URL) (substrate.AsyncMessageSource, error) {
 		conf.Insecure = false
 	}
 
-	/*
-		switch q.Get("offset") {
-		case "newest":
-			conf.Offset = OffsetNewest
-		case "oldest":
-			conf.Offset = OffsetOldest
-		case "":
-		default:
-			return nil, fmt.Errorf("ignoring unknown offset value '%s'", q.Get("offset"))
-		}
-	*/
+	switch q.Get("offset") {
+	case "newest":
+		conf.Offset = OffsetNewest
+	case "oldest":
+		conf.Offset = OffsetOldest
+	case "":
+	default:
+		return nil, fmt.Errorf("ignoring unknown offset value '%s'", q.Get("offset"))
```

Comment:
I would just change this to as we are not ignoring anything
```suggestion
		return nil, fmt.Errorf("unknown offset value '%s'", q.Get("offset"))
```